src/app/pages/SearchResultsPage/SearchResultsList.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react'
+import { Network } from '../../../types/network'
+import { useTranslation } from 'react-i18next'
+import { ResultsGroup } from './ResultsGroup'
+import { BlockDetailView } from '../BlockDetailPage'
+import { RouteUtils } from '../../utils/route-utils'
+import { TransactionDetailView } from '../TransactionDetailPage'
+import { AccountDetailsView } from '../AccountDetailsPage'
+import { SearchQueries } from './hooks'
+
+export const SearchResultsList: FC<{
+  network: Network
+  searchQueries: SearchQueries
+  roseFiatValue: number | undefined
+}> = ({ network, searchQueries, roseFiatValue }) => {
+  const { t } = useTranslation()
+  return (
+    <>
+      <ResultsGroup
```

Comment:
ResultsGroup is now no longer a great name. It groups by result type.
But ResultsOnNetwork and ResultsOnForeignNetwork (and SearchResultsList) now also group by network.

Reply:
OK, it became `ResultGroupByType`. (Also,
 - `ResultsOnNetwork` --> `ResultsOnNetworkThemed`
 - `ResultsOnForeignNetwork' -> `ResultsOnForeignNetworkThemed`
 - `SearchResultList` -> `ResultsOnNetwork`.